.github/workflows/main.yml

```diff
@@ -3,7 +3,14 @@
 # See: https://google.github.io/oss-fuzz/getting-started/continuous-integration/
 
 name: CIFuzz
-on: [pull_request]
+on:
+  pull_request:
+    paths:
+      - 'meson.build'
```

Comment:
I wonder if it would be better to use `**/meson.build` instead. In theory any change to the build system can break the fuzz targets. I'd also probably add `meson_options.txt` just in case. Other than that, the PR looks good to me.

Reply:
Makes sense, thanks!

Reply:
One last thing, as long as we don't use negative patterns I think it would make sense to sort the paths alphabetically.